torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -827,10 +964,10 @@ void Reducer::sync_bucket_indices(
   // Copy CPU tensor to device tensor, as the process_group_ could be NCCL and
   // it can only broadcast device tensors.
   auto indices_tensor_device = at::empty({total_size + 1}, options);
-  indices_tensor_device.copy_(indices_tensor, true);
+  indices_tensor_device.copy_(indices_tensor, /*non_blocking=*/true);
```

Comment:
thanks for making this more readable.

Reply:
heh, did that for my own convenience, and decided to leave it.  if it makes the call more readable for me it may do the same for someone else...